bin/hipconfig

```diff
@@ -87,11 +87,11 @@ $HSA_PATH=$ENV{'HSA_PATH'} // "$ROCM_PATH/hsa";
 $HIP_CLANG_PATH=$ENV{'HIP_CLANG_PATH'} // "$ROCM_PATH/llvm/bin";
 
 #---
-#HIP_PLATFORM controls whether to use NVCC or HCC for compilation:
-$HIP_PLATFORM=$ENV{'HIP_PLATFORM'};
```

Comment:
Shouldn't `$HIP_PLATFORM` still be read even from the environment? For example, if I installed both `hcc` and `nvcc`, it should be possible to choose at runtime which one of them should be used by HIP.

Reply:
hcc is deprecated, I assume you mean vdi/rocclr?

Reply:
I mean whatever `HIP_PLATFORM=hcc` (which appears to be the default) means...